### DISCUSSION TITLE
input-field (formated number € ) in interator-container does not respond to size changes

### DISCUSSION THREAD

**Discussion by HFolkertH** [cincheo/daquota#134] [Q&A]
hi Renaud,

The Amount field (formated number)  does not resppnd to size changes. The Description field responds okay

![image](https://user-images.githubusercontent.com/22573455/216824981-969ec548-30b1-4304-aca6-ff233cc82939.png)

**Comment by renaudpawlak** [comment]
I don't see anything wrong with your screenshot. The default size looks ok for the amount field. You should use a sm size to make it small.

**Reply by HFolkertH** [reply]
Hi Renaud,

thats the problem. This amount field  does not respond  to sm or lg in the size property. It stays default size!
